python/py-mysqlclient/Portfile

```diff
@@ -73,6 +91,9 @@ if {${name} ne ${subport}} {
             ![variant_isset mysql8] &&
             ![variant_isset mariadb55] &&
             ![variant_isset mariadb10_2] &&
+            ![variant_isset mariadb10_3] &&
+            ![variant_isset mariadb10_4] &&
+            ![variant_isset mariadb10_5] &&
             ![variant_isset percona55]} {
         default_variants +mariadb10_2
```

Comment:
I have no idea what the "best" default variant should be nowadays.  Is MariaDB 10.2 still the right choice?